server/controllers/account/register.controller.js

```diff
@@ -28,43 +28,38 @@ exports.register = function(req, res) {
   } else {
     let userLicense = new License();
     userLicense.licenseNumber = licensenum;
-    userLicense.disabled = false;
```

Comment:
userLicense.disabled = false; removed but not set as isDisabled?

Reply:
License.isDisabled gets set to false by default 😉